typescript/sdk/src/deploy/HyperlaneDeployer.ts

```diff
@@ -70,11 +71,24 @@ export abstract class HyperlaneDeployer<
   constructor(
     protected readonly multiProvider: MultiProvider,
     protected readonly factories: Factories,
-    protected readonly options?: DeployerOptions,
+    protected readonly options: DeployerOptions = {},
```

Comment:
Nit, if you're going to use a default value then you can remove the optional chaining operators below `?.`

Reply:
good spot thanks, forgot to remove the chaining